resources/views/omnomcom/orders/includes/recent-payments.blade.php

```diff
@@ -9,17 +9,21 @@
         <ul class="list-group list-group-flush">
 
             @foreach($user->withdrawals(6) as $withdrawal)
```

Comment:
Why is that 6 there?

Reply:
The 6 is because this is the recent withdrawal form and thats how the function takes in the limit